src/java/main/org/apache/zookeeper/ZooKeeper.java

```diff
@@ -127,7 +127,13 @@
  * EventNone and state sKeeperStateDisconnected.
  *
  */
-public class ZooKeeper {
+// We suppress the "try" warning here because the close() method's signature
```

Comment:
I used // comments to clarify the separation between this comment and the javadoc (since I believe the information is only important when compiling zookeeper). Let me know if you think that is valid.